hive_generator/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   build: ^2.0.0
   source_gen: ^1.0.0
   hive: ^2.0.4
-  analyzer: ">=1.0.0 <5.0.0"
+  analyzer: ">=5.0.0"
```

Comment:
Can you please also increase major version of the `hive_generator` plugin (to `2.0.0`)?

Reply:
Yes, I got it.